src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -35,6 +35,8 @@ public void isSamePerson() {
         editedAlice = new PersonBuilder(ALICE).withName(VALID_NAME_BOB).build();
         assertFalse(ALICE.isSamePerson(editedAlice));
 
+        //TODO: may have remove this tests as its no longer false
```

Comment:
Should this test case be removed?

Reply:
I'm not too sure as the test case test that a person with an all lower case name or with trailing white space is different from a capitalise name(eg Bob is not equals to bob or bob   ). However, after my implementation, the name typed into the cli will always be converted to a capitalise format, so the tests cases will always be true

Reply:
hmm, then i think we should just delete this test case